proto/vm/vm.proto

```diff
@@ -151,6 +151,7 @@ message Handler {
 
 message BuildBlockRequest {
   optional uint64 p_chain_height = 1;
+  optional bytes randomness_seed = 2;
```

Comment:
Does specifying optional do anything here? Or is this just for consistency with `p_chain_height`?

Reply:
I used "optional" so that the serialized requests would remain backward compatible. Isn't that the case ?

Reply:
It isn't clear to me that `optional bytes` is different at all from just `bytes`. See the proto spec: https://protobuf.dev/programming-guides/proto3/#field-labels